pkg/simulation/config/serviceentry.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+import (
+	networkingv1alpha3 "istio.io/api/networking/v1alpha3"
+	"istio.io/client-go/pkg/apis/networking/v1alpha3"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/howardjohn/pilot-load/pkg/simulation/model"
+)
+
+type ServiceEntrySpec struct {
+	App               string
+	AppType           model.AppType
+	Namespace         string
+	SEResolutionIndex int
```

Comment:
nit: no need to prefix "SE" when we know its SE already

Reply:
Done